blocks/banner/banner.js

```diff
@@ -0,0 +1,10 @@
+export default async function decorate(block) {
+  block.display = 'none';
```

Comment:
Do not need first and last line. it took cared already in decorateBlock it self. 
Tested locally working fine without first and last line

Reply:
okay removing this